### ISSUE TITLE
reset button to start

### ISSUE THREAD

**Issue by jhongyi** [UdaySravanK/RNSwipeButton#25]
When I got successful to swipe the end, but maybe my API response occur error, I want to reset the button to the start.
How can I do?

**Comment by cylim** [comment]
`shouldResetAfterSuccess={true}`
then set `disabled` to true, wait for your API response, if failed, set disabled to false.

**Comment by jhongyi** [comment]
> shouldResetAfterSuccess

@cylim Can I use ref to call method to reset the swipeButton?

**Comment by UdaySravanK** [comment]
@jhongyi This issue has been fixed in v1.3.0. You can use new prop `forceReset`. Please check and close this issue. Thanks

This is how you can use the new prop.

```
import React from 'react';
import {View, Button} from 'react-native';
import SwipeButton from 'rn-swipe-button';


const YourComponent = () => {
  let forceResetLastButton = null;

  return (
        <View style={styles.container}>
          <SwipeButton
            forceReset={ reset => {
              forceResetLastButton = reset
            }}
          />
          <View style={{ alignItems: 'center', marginBottom: 5 }}>
            <Button onPress={() => forceResetLastButton && forceResetLastButton()} title="Force reset" />
          </View>
        </View>
  );
};
```
